src/twisted/internet/test/test_inlinecb.py

```diff
@@ -128,20 +128,61 @@ class ForwardTraceBackTests(SynchronousTestCase):
 
     def test_forwardTracebacks(self):
         """
-        Test that chained inlineCallbacks are properly forwarding the traceback informations
-        from generator to generator
-        Reproducing #9175
+        Chained inlineCallbacks are forwarding the traceback informations
+        from generator to generator.
+
+        A first simple test with a couple of inline callbacks.
```

Comment:
I don't know if this phrase add any useful info.

I think that the docstring should be

Chained inlineCallbacks are forwarding the traceback information from generator to generator and the whole chain is available as part of traceback.